scripts/gen_mmu.py

```diff
@@ -0,0 +1,538 @@
+
+    struct_mmu_regions_format = endian_string + "III"
+    header_values_format = endian_string + "II"
+    write_4byte_bin = endian_string + "I"
+
+
+def format_string(input_str):
+    output_str = '{0: <5}'.format(str(input_str))
+    return(output_str)
+
+def pde_verbose_output(pde, binary_value):
+    if args.list_mmu_table == False:
+        return
+
+    global print_string_pde_list
+
+    present = format_string(binary_value & 0x1 )
+    read_write = format_string((binary_value >> 1 ) & 0x1 )
+    user_mode = format_string((binary_value >> 2 ) & 0x1 )
+    pwt = format_string((binary_value >> 3 ) & 0x1 )
+    pcd = format_string((binary_value >> 4 ) & 0x1 )
+    a = format_string((binary_value >> 5 ) & 0x1 )
+    ignored1 = format_string(0)
+    ps = format_string((binary_value >> 7 ) & 0x1 )
+    ignored2 = format_string(0000)
+    page_table_addr = format_string(hex((binary_value >> 12 ) & 0xFFFFF) )
+
+    print_string_pde_list += ( format_string(str(pde))+" | "+(present)+ " | "+\
+          (read_write)+ " | "+\
+          (user_mode)+ " | "+\
+          (pwt)+ " | "+\
+          (pcd)+ " | "+\
+          (a)+ " | "+\
+          (ps)+ " | "+
+          page_table_addr +"\n"
+    )
+
+def pde_print_elements():
+    global print_string_pde_list
+    print("PAGE DIRECTORY ")
+    print(format_string("PDE")+" | "+ \
+          format_string('P')  +" | "+  \
+          format_string('rw')   +" | "+  \
+          format_string('us')  +" | "+  \
+          format_string('pwt')  +" | "+  \
+          format_string('pcd')  +" | "+  \
+          format_string('a')  +" | "+   \
+          format_string('ps')  +" | "+  \
+          format_string('Addr page table'))
+    print(print_string_pde_list)
+    print("END OF PAGE DIRECTORY")
+
+
+def pte_verbose_output(pde, pte, binary_value):
+    global pde_pte_string
+
+    present    = format_string( str((binary_value >> 0) & 0x1))
+    read_write = format_string( str((binary_value >> 1) & 0x1))
+    user_mode  = format_string( str((binary_value >> 2) & 0x1))
+    pwt = format_string( str((binary_value >> 3) & 0x1))
+    pcd = format_string( str((binary_value >> 4) & 0x1))
+    a   = format_string( str((binary_value >> 5) & 0x1))
+    d   = format_string( str((binary_value >> 6) & 0x1))
+    pat = format_string( str((binary_value >> 7) & 0x1))
+    g   = format_string( str((binary_value >> 8) & 0x1))
+    alloc  = format_string( str((binary_value >> 9) & 0x1))
+    custom = format_string( str((binary_value >> 10) & 0x3))
+    page_table_addr = format_string( str(hex((binary_value >> 12) & 0xFFFFF)))
+
+    print_string_list = ( format_string(str(pte))+" | "+(present)+ " | "+\
+          (read_write)+ " | "+\
+          (user_mode)+ " | "+\
+          (pwt)+ " | "+\
+          (pcd)+ " | "+\
+          (a)+ " | "+\
+          (d)+ " | "+\
+          (pat)+ " | "+\
+          (g)+ " | "+\
+          (alloc)+ " | "+\
+          (custom)+ " | "+\
+          page_table_addr +"\n"
+    )
+    print(pde)
```

Comment:
I'm pretty sure this is what is spamming the console.
This print statement, if it is useful, should provide more surrounding context and not be printed out by default unless the user wants verbose output.